test/access_control.sh

```diff
@@ -248,8 +293,8 @@ create_user_and_assign_to_projects $LOG_USER2 $LOG_PW2 access-control-2 access-c
 oc login --username=system:admin > /dev/null
 oc project ${LOGGING_NS} > /dev/null
 
-test_user_has_proper_access $LOG_NORMAL_USER $LOG_NORMAL_PW access-control-1 access-control-2 -- access-control-3
-test_user_has_proper_access $LOG_USER2 $LOG_PW2 access-control-2 access-control-3 -- access-control-1
+test_user_has_proper_access $LOG_NORMAL_USER $LOG_NORMAL_PW access-control-1 access-control-2 access-control-3
+test_user_has_proper_access $LOG_USER2 $LOG_PW2 access-control-2 access-control-3 access-control-1
```

Comment:
What do `access-control-{1,2,3}` mean? These access for user access control would benefit from proper semantic names.

Reply:
users are assigned different projects, and access is checked depending on the which project a user has